src/js/stores/AppsStore.js

```diff
@@ -115,6 +116,14 @@ function getAppHealthWeight(health) {
   }, 0);
 }
 
+function getAppType(app) {
```

Comment:
I know this is not your logic, but let us simplify this, we only have DEFAULT and DOCKER for now and long time in the future.

The AppTypes.js should just return an object, for much better readability:
```javascript
const AppTypes = {
  DEFAULT: "DEFAULT",
  DOCKER: "DOCKER"
};
```

then this getAppType-function could be much simpler and more explicit:
```javascript
function getAppType(app) {
  if (app.container != null && app.container.type === AppTypes.DOCKER) {
    return AppTypes.DOCKER;
  }
  return AppTypes.DEFAULT;
}
```
Even if we should add another type, this should be more readable.